content-scripts/components/HeaderAuthentication.tsx

```diff
@@ -0,0 +1,138 @@
+import jsx from "texsaur";
+import { AuthSession } from "../types";
+import { togglePopover } from "../modules/utilities/popover";
+
+interface Props {
+  auth: AuthSession | null;
+}
+
+const Authentication = ({ auth }: Props) => {
+  if (auth)
+    return (
+      <div id="se-auth">
+        <button
+          id="se-auth-notifications-button"
+          className={`se-button se-icon-button ${auth.hasNotifications ? "se-badge" : ""
+            }`}
+          onclick={() => togglePopover("se-auth-notifications-menu")}
+        >
+          <span className="se-icon ri-notification-line"></span>
+        </button>
+        <div id="se-auth-notifications-menu">
+          <input
+            type="radio"
+            name="se-auth-notifications"
+            id="se-auth-new-notifications-radio"
+            checked
+          />
+          <input
+            type="radio"
+            name="se-auth-notifications"
+            id="se-auth-read-notifications-radio"
+          />
+          <div id="se-auth-notifications-header">
+            <label htmlFor="se-auth-new-notifications-radio">Novas</label>
+            <label htmlFor="se-auth-read-notifications-radio">Lidas</label>
+          </div>
+          <div id="se-auth-notifications-list">
+            <ol id="se-auth-new-notifications"></ol>
+            <ol id="se-auth-read-notifications"></ol>
+            <div id="se-auth-empty-notifications">
+              <span className="se-icon ri-notification-off-line"></span>
+              <span>Sem notificações</span>
+            </div>
+            <div className="se-loading-indicator">
+              <span className="se-icon ri-refresh-line"></span>
+            </div>
+          </div>
+        </div>
+
+        <button id="se-auth-profile-button" onclick={() => togglePopover("se-auth-profile-menu")}>
+          <img
+            src={`fotografias_service.foto?pct_cod=${auth.number}`}
+            alt="Foto de perfil"
+          />
+        </button>
+        <div id="se-auth-profile-menu">
+          <div id="se-auth-header">
+            <span>{auth.name}</span>
+            <span>{auth.number}</span>
+          </div>
```

Comment:
This could also be a good opportunity to make this clickable. Personally, I always try to click it...

Reply:
The all header?

Reply:
I don't think that would be a good ux

Reply:
Yeah, it's most likely just me being used to other websites, not necessarily a general issue.